release/serve_tests/workloads/microbenchmarks.py

```diff
@@ -242,8 +286,31 @@ async def _main(output_path: Optional[str]):
 
 @click.command()
 @click.option("--output-path", "-o", type=str, default=None)
-def main(output_path: Optional[str]):
-    asyncio.run(_main(output_path))
+@click.option("--run-all", is_flag=True)
+@click.option("--run-http", is_flag=True)
+@click.option("--run-grpc", is_flag=True)
+@click.option("--run-handle", is_flag=True)
+@click.option("--run-latency", is_flag=True)
+@click.option("--run-throughput", is_flag=True)
```

Comment:
what's the default?

Reply:
defaults are all False

Reply:
seems confusing; would suggest default runs everything (or the most common one in isolation)

Reply:
Added logic that defaults to run everything if no other flags are set!